### DISCUSSION TITLE
Model-viewer is loading but isn't showing any model, and getting an error in the console while opening the same link from the windows

### DISCUSSION THREAD

**Discussion by abhishekmeena33** [google/model-viewer#4819] [Q&A]
I am simply hosting my model-viewer on github pages but it couldn't load any model, but it was in the morning, idk what should i do, I am getting this error in the inspector of chrome when opened the link fro my windows. I have used a model less than size 20mb, do i need to run **git lfs install** and **git lfs pull** and then run it. I have done it, but it is not working, or am i not doing it correctly? Please help.
![image](https://github.com/0hq/WebGPT/assets/118628637/14381eed-7098-412b-9847-1b9ea004a4d7)

**Comment by elalish** [comment]
I think you should ask on Github's forums - sounds like they're giving you the pointer file instead of the actual GLB. I don't know a whole lot about LFS.
